gpib_server.py

```diff
@@ -177,8 +177,10 @@ def read(self, c, n_bytes=None):
         """
         instr = self.getDevice(c)
         if n_bytes is None:
+            instr.timeout = c['timeout']['ms']
```

Comment:
No reason to have this inside the `if` statement, since it's reproduced in the `else`. Just refactor to a single line before the `if/else`.